extension/agenthealth/handler/stats/provider/statuscode.go

```diff
@@ -111,8 +111,8 @@ func (h *StatusCodeHandler) updateStatusCodeCount(stats *[5]int, statusCode int,
 
 func GetShortOperationName(operation string) string {
 	switch operation {
-	case "PutMetricData":
```

Comment:
pmd being removed since we already get status codes for pmd/ple/pts.

For PMD:
[tiny.amazon.com/ut6mtcin/iGraph](https://tiny.amazon.com/ut6mtcin/iGraph)
For PLE:
[tiny.amazon.com/nah7tlqw/iGraph](https://tiny.amazon.com/nah7tlqw/iGraph)